.NET/Microsoft.Recognizers.Definitions.Common/English/DateTimeDefinitions.cs

```diff
@@ -190,6 +190,7 @@ public static class DateTimeDefinitions
       public static readonly string TimeNumberCombinedWithUnit = $@"\b(?<num>\d+(\.\d*)?){TimeUnitRegex}";
       public static readonly string[] BusinessHourSplitStrings = { @"business", @"hour" };
       public const string NowRegex = @"\b(?<now>(right\s+)?now|as soon as possible|asap|recently|previously)\b";
+      public static readonly string NowDateRegex = $@"\b({NowRegex}|^(date)$)\b";
```

Comment:
Perhaps rename this to "NowParseRegex"?
Also, wouldn't it be best to treat "^(date)$" as a named group to makes the parse behaviour explicit, as in onewordperiodregex? I suggest adding a comment on the YAML that this component wouldn't need to be translated .

Reply:
Renamed regex and added comment.
But I do not see the reason to use a named group here, "date" does not require a specific parse processing, it is handled as the other elements of NowRegex.

Reply:
Not a major issue. The point was to clarify for those localizing the regexes to other languages that this is English-specific and not exactly the same as "now".

Reply:
I'm fine with merging as-is once conflicts are resolved.